editor/gui/editor_file_dialog.h

```diff
@@ -239,6 +239,8 @@ class EditorFileDialog : public ConfirmationDialog {
 	static void _bind_methods();
 
 public:
+	Color get_dir_icon_color(String dir_path);
```

Comment:
Are you sure? It can't be `const` because I'm modifying it inside the method. The same goes for the `&` – I'm modifying it, should it really be a reference?

Reply:
You're not modifying it? And you shouldn't

Reply:
Oh, yea, you are right. It was the previous piece of this function that I've eventually removed that did modify the argument.

Reply:
Generally you should avoid modifying arguments 🙂